README.md

```diff
@@ -44,6 +46,51 @@ The data will be sent formatted like this:
 - `bazar_form_id`: the ID of the Bazar item
 - `data`: all the data of the Bazar item
 
+### ActivityPub
+
+The ActivityPub format only works if the Bazar objects are semantically defined. Please see [the documentation](https://yeswiki.net/documentation/?BazarSemantique) (in French) to know how to do that. You must use `https://www.w3.org/ns/activitystreams` as the main context, but other contexts can be added as well.
+
+The data will be formatted like this:
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type":"Create",
+  "to": [
+    "http://localhost/?WikiAdmin/followers",
+    "https://www.w3.org/ns/activitystreams#Public"
+  ],
+  "actor": "http://localhost/?WikiAdmin",
+  "object": { 
+    "id": "http://localhost/?MyComment",
+    "type": "Note",
+    "attributedTo": "http://localhost/?WikiAdmin",
+    "to": [
+      "http://localhost/?WikiAdmin/followers",
+      "https://www.w3.org/ns/activitystreams#Public"
+    ],
+    "title":"My comment",
+    "content":"Thank you for reading my comment",
+    "published":"2019-10-21T14:56:32Z",
+    "updated":"2019-10-21T14:56:32Z"
+  }
+}
+```
+
+For the `Delete` activity, only the object URI is sent.
+
+The `published` and `updated` fields are set automatically, unless you mark a field explicitely with this context.
+
+#### ActivityPub actors
+By default, the actor posting the activity will be the logged-in user. You can also mark a field with the context `https://www.w3.org/ns/activitystreams#author` to override this value.
+
+You can set two parameters in the `wakka.config.php` file :
+
+- `webhooks_activitypub_default_actor`: The URL you associate with this parameter will be used as an actor for **all** ActivityPub activities.
+
+- `webhooks_activitypub_actors_base_url`: The base URL to use for all actors. For example if you use `https://activitypub.server/actor/`, then the URI of user WikiAdmin will be `https://activitypub.server/actor/WikiAdmin` instead of `http://localhost/?WikiAdmin`.
+
```

Comment:
Maybe we can consider an actor called Anonymous that would be author of all non identified contributions?